ts/types/Search.ts

```diff
@@ -4,3 +4,10 @@ export type SearchOptions = {
   noteToSelf: string;
   isSecondaryDevice: boolean;
 };
+
+export type AdvancedSearchOptions = {
+  query: string;
+  from: string;
```

Comment:
```suggestion
  from: string?;
```
This should be optional

Reply:
Wouldn't it be 
```
from?:string
```
?

Reply:
Oh yes, you're right